lib/PlaidClient.js

```diff
@@ -490,6 +490,20 @@ Client.prototype.createPayment =
     }, cb);
   };
 
+// createPaymentWithOptions(String, String, Object, Object, Function)
+Client.prototype.createPaymentWithOptions =
```

Comment:
It looks like the way we typically do options (above) is to pass it into `_authenticatedRequest`. Is a separate function just to avoid a breaking change?

Reply:
The separate function is to avoid the breaking change by adding a new arg to `createPayment`.  Thanks for pointing out the pattern using `_authenticatedRequest`.  I think I can do something similar and juggle the params around in `createPayment` to avoid creating a new function and a breaking change.